src/js/store/index.ts

```diff
@@ -52,6 +54,7 @@ const getDefaultState = Record<LyraState>({
       widgets: Map<string, WidgetRecord>(),
       signals: defaultSignalState,
       layouts: Map<string, LayoutRecord>(),
+      facetLayouts: Map<string, FacetLayoutRecord>(),
```

Comment:
suggestion per our conversation today: since there can only be one vega layout spec, change this from a Map to a single FacetLayoutRecord

Reply:
Probably just not understanding typescript syntax/types here, but when I change the line to `facetLayouts: FacetLayoutRecord` I get an error:`'FacetLayoutRecord' only refers to a type, but is being used as a value here.`

Reply:
Here you're initializing the actual values of a record, not defining an interface. So what you're trying to do in this line is set the value of a property `facetLayouts` to a default FacetLayoutRecord. So it should be `facetLayouts: FacetLayoutRecord()` -- note the parentheses for initialization

Reply:
hmm I'm getting the same error even with the parentheses

Reply:
i meant `FacetLayout()`, for the constructor and not the type

Reply:
ohhh make sense.  realizing with the proposed change to facet layouts to make it compatible with the other layout mechanism, we could in theory support multiple facets and might want to keep it as a map after all?